### ISSUE TITLE
Some devices should be filtered in "Add host devices" dialog

### ISSUE THREAD

**Issue by yunmingyang** [cockpit-project/cockpit-machines#605]
Version:
libvirt-dbus-1.4.0-4.fc35.x86_64
cockpit-264-1.fc35.x86_64
cockpit-machines-260-1.fc35.noarch

Description:
As passing through some host devices may cause some terrible mistakes, we should limit some devices so that they will not in "Add host devices" dialog. After checking how virt-manager to do that, I found that: https://github.com/virt-manager/virt-manager/blob/main/virtManager/addhardware.py#L769 and https://github.com/virt-manager/virt-manager/blob/main/virtManager/addhardware.py#L771. I think they filter both the usb devices whose vendor id is "0x1d6b" and product id is one of "0x0001", "0x0002", "0x0003", and the pci devices whose capability type is pci_bridge. So maybe we should also do that

**Comment by skobyda** [comment]
I don't think this is a duplicate. The other issue seems to be related to attachment of IOMMU groups, while this seems to be relateed to attachmend of devices with id 0x0001/2/3, which according to http://www.linux-usb.org/usb.ids are **root hubs**. I personally recommend 2 different approaches to these 2 different problems.
Indeed root hubs should probably not be attachable, and therefore we could filter them out (or disable them)
